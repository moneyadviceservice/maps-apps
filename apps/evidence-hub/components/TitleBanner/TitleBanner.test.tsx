import { render, screen } from '@testing-library/react';

import { TitleBanner } from '.';

describe('TitleBanner component', () => {
  it('renders correctly', () => {
    render(<TitleBanner title="TitleBanner" />);
    const banner = screen.getByTestId('title-banner');
    expect(banner).toMatchSnapshot();
  });
});
