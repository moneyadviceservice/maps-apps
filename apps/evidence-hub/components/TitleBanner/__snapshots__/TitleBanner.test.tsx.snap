// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBanner component renders correctly 1`] = `
<div
  class="bg-blue-700"
  data-testid="title-banner"
>
  <div
    class="container-auto max-w-[1272px] py-4"
  >
    <h1
      class="text-4xl md:text-6xl font-bold text-white"
    >
      TitleBanner
    </h1>
  </div>
</div>
`;
